packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -255,6 +255,11 @@ class SwipeableViews extends Component {
      */
     disabled: PropTypes.bool,
     /**
+     * If `true`, it will enable mouse events.
+     * This will allow the user to perform the relevant swipe actions with a mouse.
+     */
+    enableMouseEvents: PropTypes.bool,
```

Comment:
Can we add a default value for that property? Just for the documentation purposes.